### DISCUSSION TITLE
Option to customize default landing page after logging in

### DISCUSSION THREAD

**Discussion by Bottlecapps** [apache/superset#18691] [Ideas]
We have about 400 users using our Dashboards built on superset. One of the biggest pain point is user or admin cannot customize the default page that a user goes to after logging in. Lot of our user community logs into Superset to view their respective dashboard. It would be good if we can take them directly to their Dashboard page instead of taking them to the home page.

**Comment by samarsodhi** [comment]
Any update? this seems like a powerful feature which will help with providing better user experience.i.e.making superset more customizable for users will reduce the number of clicks required to get to what users want.
